lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -601,6 +602,24 @@ emptyByronWalletWith ctx style (name, mnemonic, pass) = do
     expectResponseCode @IO HTTP.status201 r
     return (getFromResponse id r)
 
+emptyByronWalletFromXPrvWith
+    :: forall t. ()
+    => Context t
+    -> String
+    -> (Text, Text, Text)
```

Comment:
What about using a less stringly-typed tuple here like: (WalletName, ByronKey 'RootK XPrv, ByteString) 

And do the actual transformation to `Text` in this function instead ?

Reply:
yeah, we can do it. but I tried to comply with what is in DSL module, most endpoint  functions use Text